frontend/webapp/core/fetch_config.py

```diff
@@ -4,6 +4,7 @@
 import requests
 from webapp.utils import API_URL_FLASK
 from webapp.utils import flatten
+from webapp.utils import translate_rfc2622
```

Comment:
added but not used?

Reply:
also, unused variable `self.raw_json_config`.. why is it not found by the pre-commit hooks?

Reply:
I will remove the unused variables. I think I had deactivated pre-commit for another reason, will re-check.